src/ducks/billsMatching/Linker/billsToOperation/operationsFilters.spec.js

```diff
@@ -31,9 +31,9 @@ describe('operations filters', () => {
       const bill = { matchingCriterias: { labelRegex: 'Bidule' } }
       const fByBrand = filterByBrand(bill)
 
-      expect(fByBrand({ label: 'Chez Bidule !!!' })).toBe(true)
-      expect(fByBrand({ label: 'bidule 20/08' })).toBe(true)
-      expect(fByBrand({ label: 'Ca match paaaas du tout' })).toBe(false)
+      expect(fByBrand({ label: 'Chez Bidule !!!' })).toBeTruthy()
+      expect(fByBrand({ label: 'bidule 20/08' })).toBeTruthy()
+      expect(fByBrand({ label: 'Ca match paaaas du tout' })).toBeFalsy()
```

Comment:
Truthy and falsy are not the same as true/false, it seems we are lowering the bar of the test here, what is the gain ?

Reply:
Actually I don't know why but these tests were passing and then they were failing. I don't know how they passed before since `fByBrand` returns the result of `label.match(regex)`, so it returns `null` or an array, but never a boolean. That's why I  used `toBeTruthy` and `toBeFalsy` to fix this. But another way would be to `return Boolean(label.match(regex))` and still use `toBe(false)` and `toBe(true)` in tests. What do you think?

Reply:
I think it is better to be fairly strict on what we return, so Boolean(label.match(regex)) is good to me.

> Be conservative in what you send, be liberal in what you accept

https://en.wikipedia.org/wiki/Robustness_principle

Reply:
Otherwise, at some point, someones sees that you return the regex result, and starts using it instead of considering it as a boolean, then you want to change the behavior and not use a regex engine, and you break the implicit API for the other person.

Reply:
:+1: